tests/test_batch.py

```diff
@@ -144,28 +133,3 @@ def test_batch_job_result_not_ready(db, col, docs):
     assert batch_db.commit() == [job]
     assert len(job.result()) == len(docs)
     assert extract("_key", col.all()) == extract("_key", docs)
-
-
-def test_batch_bad_state(db, col, docs):
```

Comment:
Note that this test no longer makes sense with the current implementation, because we don't throw this exception anymore.